docs/contribution/documentation_cheatsheet.md

```diff
@@ -18,35 +48,51 @@ A link to {class}`Acts::Volume`.
 
 * Code: 
 
-````markdown
-```{doxygenclass} Acts::Volume
+```text
+:::{doxygenclass} Acts::Volume
 ---
 members: center
 ---
+:::
 ```
-````
 
 * Result:
 
-```{doxygenclass} Acts::Volume
+:::{doxygenclass} Acts::Volume
 ---
 members: center
 ---
-```
+:::
 
 (cheatsheetlabels)=
 ## Cross referencing and labels
 
 * Setting a label:
 
-```markdown
+```text
 (cheatsheetlabels)=
```

Comment:
Maybe not that this should be in front of something, e.g. in front of a `#` headline.